core/src/main/java/org/apache/accumulo/core/client/IsolatedScanner.java

```diff
@@ -247,6 +247,9 @@ public void setTimeOut(int timeOut) {
       setTimeout(timeOut, TimeUnit.SECONDS);
   }
 
+  /**
+   * @deprecated
```

Comment:
The javadoc tool is smart enough to identify Deprecated annotations. The addition of a javadoc tag for deprecation does nothing, unless there is a comment associated with it to explain the deprecation details.